pkg/apis/sessions/session_state.go

```diff
@@ -256,3 +265,23 @@ func DecodeSessionState(v string, c *encryption.Cipher) (*SessionState, error) {
 	}
 	return ss, nil
 }
+
+func (s *SessionState) RawClaims() map[string]interface{} {
+	return s.rawClaims
+}
+
+func (s *SessionState) RawClaimsValid() bool {
+	return s.rawClaimsValid
+}
+
+func (s *SessionState) SetRawClaims(rawClaims map[string]interface{}) {
+	s.rawClaims = rawClaims
+	s.rawClaimsValid = true
+}
+
+func (s *SessionState) SetRawClaimsFromIDToken(idToken *oidc.IDToken) (err error) {
+	if err = idToken.Claims(&s.rawClaims); err == nil {
+		s.rawClaimsValid = true
+	}
+	return
+}
```

Comment:
I'm not a fan of named return variables and generally believe they are a bit of an anti-pattern. They become especially hard to track in larger methods. Could you please review the code in this PR and try to remove named returns?